crates/pop-cli/src/commands/up/contract.rs

```diff
@@ -30,49 +30,49 @@ const FAILED: &str = "🚫 Deployment failed.";
 #[derive(Args, Clone)]
 pub struct UpContractCommand {
 	/// Path to the contract build directory.
-	#[arg(short = 'p', long)]
+	#[arg(short, long)]
 	path: Option<PathBuf>,
 	/// The name of the contract constructor to call.
-	#[clap(name = "constructor", long, default_value = "new")]
+	#[clap(short, long, default_value = "new")]
 	constructor: String,
 	/// The constructor arguments, encoded as strings.
-	#[clap(long, num_args = 0..,)]
+	#[clap(short, long, num_args = 0..,)]
 	args: Vec<String>,
 	/// Transfers an initial balance to the instantiated contract.
-	#[clap(name = "value", long, default_value = "0")]
+	#[clap(short, long, default_value = "0")]
 	value: String,
 	/// Maximum amount of gas to be used for this command.
 	/// If not specified it will perform a dry-run to estimate the gas consumed for the
 	/// instantiation.
-	#[clap(name = "gas", long)]
+	#[clap(name = "gas", short, long)]
 	gas_limit: Option<u64>,
 	/// Maximum proof size for the instantiation.
 	/// If not specified it will perform a dry-run to estimate the proof size required.
-	#[clap(long)]
+	#[clap(short = 'P', long)]
 	proof_size: Option<u64>,
 	/// A salt used in the address derivation of the new contract. Use to create multiple
 	/// instances of the same contract code from the same account.
-	#[clap(long, value_parser = parse_hex_bytes)]
+	#[clap(short = 'S', long, value_parser = parse_hex_bytes)]
 	salt: Option<Bytes>,
 	/// Websocket endpoint of a chain.
-	#[clap(name = "url", long, value_parser, default_value = DEFAULT_URL)]
+	#[clap(short, long, value_parser, default_value = DEFAULT_URL)]
 	url: Url,
 	/// Secret key URI for the account deploying the contract.
 	///
 	/// e.g.
 	/// - for a dev account "//Alice"
 	/// - with a password "//Alice///SECRET_PASSWORD"
-	#[clap(name = "suri", long, short, default_value = "//Alice")]
+	#[clap(short, long, default_value = "//Alice")]
 	suri: String,
 	/// Perform a dry-run via RPC to estimate the gas usage. This does not submit a transaction.
-	#[clap(long)]
+	#[clap(short = 'D', long)]
 	dry_run: bool,
 	/// Uploads the contract only, without instantiation.
-	#[clap(short('u'), long)]
+	#[clap(short = 'U', long)]
```

Comment:
Note: this is a (minor) breaking change, required so that -u is consistent for url across calling and uploading contracts. I figure upload only is probably used rarely and therefor `may` be safe to make this change now, to resolve the inconsistencies across commands which is much more frustrating in my view.

For example, I just want to do `pop up contract -u my_rpc ...` and then `pop call contract -u my_rpc ...`